homeassistant/components/faadelays/strings.json

```diff
@@ -0,0 +1,19 @@
+{
+  "title": "FAA Delays",
```

Comment:
Does this need to be translated?

Reply:
It does, yes! Thanks for the pointer, added to translations.